oxidation/bindings/electron/src/meths.rs

```diff
@@ -21,7 +21,7 @@ fn struct_availabledevice_js_to_rs<'a>(
                 |s: String| -> Result<_, &'static str> { Ok(std::path::PathBuf::from(s)) };
             match custom_from_rs_string(js_val.value(cx)) {
                 Ok(val) => val,
-                Err(err) => return cx.throw_type_error(err),
+                Err(err) => return cx.throw_type_error(AsRef::<str>::as_ref(&err)),
```

Comment:
`AsRef::<str>::as_ref` is not needed, because `throw_type_error` take `AsRef<str>`  as argument

Reply:
done